telemetry/telemetry.proto

```diff
@@ -6,15 +6,90 @@ option java_package = "io.dronecore.telemetry";
 option java_outer_classname = "TelemetryProto";
 
 service TelemetryService {
-    rpc SubscribePosition(SubscribePositionRequest) returns(stream Position) {}
+    rpc SubscribePosition(SubscribePositionRequest) returns(stream PositionResponse) {}
+    rpc SubscribeHome(SubscribeHomeRequest) returns(stream HomeResponse) {}
+    rpc SubscribeInAir(SubscribeInAirRequest) returns(stream InAirResponse) {}
+    rpc SubscribeArmed(SubscribeArmedRequest) returns(stream ArmedResponse) {}
+    rpc SubscribeAttitudeQuaternion(SubscribeAttitudeQuaternionRequest) returns(stream AttitudeQuaternionResponse) {}
+    rpc SubscribeAttitudeEuler(SubscribeAttitudeEulerRequest) returns(stream AttitudeEulerResponse) {}
+    rpc SubscribeCameraAttitudeQuaternion(SubscribeCameraAttitudeQuaternionRequest) returns(stream CameraAttitudeQuaternionResponse) {}
+    rpc SubscribeCameraAttitudeEuler(SubscribeCameraAttitudeEulerRequest) returns(stream CameraAttitudeEulerResponse) {}
+    rpc SubscribeGroundSpeedNED(SubscribeGroundSpeedNEDRequest) returns(stream GroundSpeedNEDResponse) {}
+    rpc SubscribeGPSInfo(SubscribeGPSInfoRequest) returns(stream GPSInfoResponse) {}
+    rpc SubscribeBattery(SubscribeBatteryRequest) returns(stream BatteryResponse) {}
+    rpc SubscribeFlightMode(SubscribeFlightModeRequest) returns(stream FlightModeResponse) {}
+    rpc SubscribeHealth(SubscribeHealthRequest) returns(stream HealthResponse) {}
+    rpc SubscribeRCStatus(SubscribeRCStatusRequest) returns(stream RCStatusResponse) {}
 }
```

Comment:
How do we handle `Unsubscribe()` ? ; we pass `nullptr` in case of  C++.

Reply:
That's a good question.

I see two ways:

1. Maybe we can have an `Unsubscribe*` equivalent for each stream, which would make the stream end. I am not sure how clean that is on the server side.
2. (Currently my preferred one) we let the client unsubscribe from the stream whenever it wants, in a pubsub manner.

I checked online on how to make the client unsubscribe from a server stream in gRPC, and [found this for Java](https://github.com/grpc/grpc-java/pull/3115). It allows the client to "cancel" a server stream. I therefore assumed that it should be possible to do that from other languages, but it's right that we will need to test it more thoroughly.

Reply:
OK Cool.